en/services/flight_modes.md

```diff
@@ -0,0 +1,494 @@
+       <li>Throttle stick controls up/down speed with a predetermined maximum rate (and movement speed in other axes).</li> 
+       <li>Yaw stick controls rate of angular rotation above the horizontal plane.</li> 
+      </ul>
+    </li>
+    <li>Takeoff:
+      <ul>
+       <li>When landed, the vehicle will take off if the throttle stick is raised above 62.5% percent (of the full range from bottom).</li>
+      </ul>
+    </li>
+  </ul>
+ </p>
+ </td>
+</tr>
+
+
+<tr id="manual_stabilized_mc">
+ <td>Manual/ Stabilized</td>
+ <td>Stabilized</td>
+ <td>S</td>
+ <td>S<sub>rate</sub></td>
+ <td>M</td>
+ <td></td>
+ <td><p>RC mode where centered sticks level vehicle (only - position is not stabilized).</p>
+   <p>
+   <ul>
+    <li>Centered RP sticks level vehicle.</li>
+    <li>Outside center:
+      <ul>
+       <li>Roll/Pitch sticks control tilt angle in those orientations, resulting in corresponding left-right and forward-back movement.</li>
+       <li>Throttle stick controls up/down speed (and movement speed in other axes).</li>
+       <li>Yaw stick controls rate of angular rotation above the horizontal plane.</li>
+      </ul>
+    </li>
+    </ul>
+  <p>
+</td>
+</tr>
+
+
+<tr id="rattitude_mc">
+ <td>Rattitude</td>
+ <td>Stabilized</td>
+ <td>S or S<sub>rate</sub></td>
+ <td>S<sub>rate</sub></td>
+ <td>M</td>
+ <td></td>
+ <td>
+ <p>RC mode that allows pilots to fly using <a href="#manual_stabilized_mc">Manual/Stabilized</a> flight most of the time, but still perform <a href="#acro_mc">Acro mode</a>-style flips and tricks when desired. Centered sticks level vehicle.
+  <ul>
+    <li>Sticks within mode's threshold (like <a href="#manual_stabilized_mc">Manual/Stabilized mode</a>).
+      <ul>
+       <li>Centered RP sticks level vehicle. Roll/Pitch sticks control tilt angle in those orientations, resulting in corresponding left-right and forward-back movement.</li>
+      </ul>
+    </li>
+    <li>Sticks outside threshold (like <a href="#acro_mc">Acro mode</a>): 
+      <ul>
+       <li>RPY stick inputs control the rate of angular rotation around the respective axes.</li>
+      </ul>
+    </li>
+  </ul>
+ </p>
+</td>
+</tr>
+
+
+<tr id="acro_mc">
+ <td>Acro</td>
+ <td>Stabilized</td>
+ <td>S<sub>rate</sub></td>
+ <td>S<sub>rate</sub></td>
+ <td>M</td>
+ <td></td>
+ <td><p>RC mode for performing acrobatic maneuvers e.g. flips, rolls and loops.</p> 
+  <p>RC RPY stick inputs control the rate of angular rotation around the respective axes. Throttle is passed directly to the output mixer. When sticks are centered the vehicle will stop rotating, but remain in its current orientation (e.g. possibly inverted) and moving according to its current momentum.</p>
+ </td>
+</tr>
+
+
+<tr id="orbit_mc">
+ <td>Orbit</td>
+ <td>Guided</td>
+ <td>-</td>
+ <td>-</td>
+ <td>-</td>
+ <td></td>
+ <td><p>GCS-initiated guided mode for flying a circle, always facing the center.</p> 
```

Comment:
It is desirable for this to map to ArduPilot [Circle mode](http://ardupilot.org/copter/docs/circle-mode.html). Both modes circle facing inwards, and both will automatically do so without user intervention. 

Differences are that circle mode allows altitude and yaw to be changed with sticks, default rotation speed and direction is set by parameter (rather than hard coded initial value with stick override).

Can these be somehow treated as one, or are these differences too great (IMO yes, too different). These are like sub modes.